src/deepsparse/utils/onnx.py

```diff
@@ -494,3 +496,45 @@ def default_cached_outputs(model_path: str) -> List[bool]:
     assert len(output_names) > 0
 
     return [name.startswith(CACHE_OUTPUT_PREFIX) for name in output_names]
+
+
+def has_model_kv_cache(model: Union[str, ModelProto]) -> bool:
+    """
+    Check whether a model has a KV cache support.
+
+    :param model_path: Path to a model or a model proto.
+    :return True if the model has a KV cache support, False otherwise.
+    """
+    return bool(any(default_cached_outputs(model)))
+
+
+def overwrite_cache_model_inputs(
+    model_path: str,
+    input_ids_length: int,
+    sequence_length: int,
+) -> Tuple[str, int]:
```

Comment:
This just returns `str` now

Reply:
Although honestly it might help if we returned the same stuff as `overwrite_onnx_model_inputs_for_kvcache_models` as I think a lot of code needs to use the cached outputs anyway when generating, which happens right after overwriting

Reply:
(at least I use them 😛 )